lib/kubernetes-deploy/resource_watcher.rb

```diff
@@ -41,14 +41,19 @@ def run(delay_sync: 3.seconds, reminder_interval: 30.seconds)
     def report_what_just_happened(resources, watch_time)
       new_successes, new_failures = resources.partition(&:deploy_succeeded?)
       new_failures.each do |resource|
+        tags = %W(namespace:#{@namespace} sha:#{ENV['REVISION']} resource:#{resource.id})
```

Comment:
Collisions on names like `web` (ingress vs pod) are pound to happen but fair comment about type. That seems useful, just pondering the cost vs value at this point